hyperSpec/R/attach_hySpc.R

```diff
@@ -0,0 +1,23 @@
+#' Load all installed packages of the hySpc.\* package family
+#'
+#' This function loads all installed packages of the **hySpc._something_**
+#' package family.
+#'
+#' @param dont_attach Packages that should *not* be attached
+#' @param ... further parameters are handed to [base::library()]
+#'
+#' @export
+#' @importFrom utils installed.packages
+#' @examples
+#' attach_hySpc()
+attach_hySpc <- function(dont_attach = "hySpc.testthat", ...){
```

Comment:
1. I recommend following the **tidyverse** style code and use all lower case letters in **function** names.
2. I'd prefer the word `load` over `attach`. Yet `attach` would be a more precise term (as packages are loaded and attached). 
3. I think it's better to have several functions of this type. Related to this functionality and they should have the common prefix `hyspc_`, e.g (just ideas to illustrate my point):
    - `hyspc_load_all()` or `hyspc_attach_all()`
    -  `hyspc_load_main()`
    -  `hyspc_load_manipulation()`
    -  `hyspc_load_tidyverse()` (ggplot + dplyr)
    -  `hyspc_load_read()`
    -  `hyspc_packages()` 
    -  `hyspc_update()`
    -  `hyspc_generate_installation_code()`

    Even if we decide to have only a single function, I'd prefer `hyspc_load()` over `load_hyspc()` or `hyspc_attach()` over `attach_hyspc()`

Reply:
I put this into the issue since I think it deserves a wider discussion.